NOTICES

```diff
@@ -8,7 +8,6 @@ Distribution: EXTERNAL
 
 Components: 
 
-@apollo/client 3.3.3 : MIT License
```

Comment:
I’m confused, we still depend on this in Navigator. Why did this get removed?

Reply:
As the next line says: this is a change I've made specifically in the hope that the CI run will see the file as changed, and therefore will open a PR to change it back.

I'm assuming that the current NOTICES file we have is the exact result of the process described in the NOTICES.md file per #8161, but test runs will confirm (o infirm) that.

Note that I have not run anything locally.

Reply:
duh, sorry I missed the next line.